server/apis/v1/handler.go

```diff
@@ -469,13 +469,26 @@ func (h *handler) UpdateInterStepBufferService(c *gin.Context) {
 
 // DeleteInterStepBufferService is used to update the spec of the inter step buffer service
 func (h *handler) DeleteInterStepBufferService(c *gin.Context) {
-	ns, isbServices := c.Param("namespace"), c.Param("isb-services")
+	ns, isbsvcName := c.Param("namespace"), c.Param("isb-service")
 
-	// TODO: validate if it's being used by any pipelines
-	err := h.numaflowClient.InterStepBufferServices(ns).Delete(context.Background(), isbServices, metav1.DeleteOptions{})
+	pipelines, err := h.numaflowClient.Pipelines(ns).List(context.Background(), metav1.ListOptions{})
+	// Get(context.Background(), pipeline, metav1.GetOptions{})
```

Comment:
do we need this?

Reply:
The pipeline spec has the ISBSVC name it uses... I use this information to determine if the ISBSVC can be deleted or not.